.github/workflows/main.yml

```diff
@@ -16,7 +16,9 @@ jobs:
       image: python:${{ matrix.python }}
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

Comment:
Why is it here? 
If you needed to change version do it please in seperate PR

Reply:
I do not know... it is now changed back to v2